instrumentation/src/com/intellij/rt/coverage/instrumentation/Instrumenter.java

```diff
@@ -156,4 +159,13 @@ public boolean hasInterfaces() {
   public boolean isKotlinClass() {
```

Comment:
it's also Kotlin reference, I wish we can store annotations and pass them to filters later.